lib/galaxy/config.py

```diff
@@ -276,6 +276,11 @@ def __init__( self, **kwargs ):
         # On can mildly speed up Galaxy startup time by disabling index of help,
         # not needed on production systems but useful if running many functional tests.
         self.index_tool_help = string_as_bool( kwargs.get( "index_tool_help", True ) )
+        self.tool_name_boost = kwargs.get( "tool_name_boost", None )
```

Comment:
Unless we're suggesting a major departure from previous practice, the attribute in config reflects what's represented in the galaxy config, whether it's the default value or the one a user has set through manipulation.  I can't see that changing this will be anything but confusing.  If a component *wants* to override these defaults, it still can.

As this is written, if you wanted to use these (app-wide) configuration values elsewhere, you'd have to re-specify all the defaults with *even more* app.config.get('*_boost', VALUE) statements, resulting in a lot of duplication.